testutils/clustermanager/gke_test.go

```diff
@@ -739,7 +741,10 @@ func TestAcquire(t *testing.T) {
 			}, nil, false,
 		}, {
 			// cluster creation succeeded
-			nil, "", false, []string{}, []string{}, &container.Cluster{
+			nil, "", false, []string{}, []string{}, true, nil, nil, false,
```

Comment:
Shouldn't this be two separate test cases, one with skip and one without?

Reply:
This one is false, and the testcase below is  true. Maybe I didn't get your question correctly, so leave it up for you to resolve